kuma/users/stripe_utils.py

```diff
@@ -67,3 +70,21 @@ def retrieve_stripe_subscription_info(user):
         }
 
     return None
+
+
+def create_missing_stripe_webhook():
+    url_path = reverse("users.stripe_payment_succeeded_hook")
+    url = (
+        settings.WEBHOOK_HOSTNAME + url_path
```

Comment:
As discussed, this would fail if `settings.STRIPE_WEBHOOK_HOSTNAME == 'abc123.ngrok.io'`